### PR TITLE
Migrate from Godeps to go modules

### PR DESCRIPTION
**Pull request by mendelgusmao**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module github.com/registrobr/rdap-client
+
+go 1.14
+
+require (
+	github.com/aryann/difflib v0.0.0-20121123051704-035af7c09b12
+	github.com/codegangsta/cli v1.19.2-0.20170424213122-ab403a54a148
```
Comment (rafaeljusto) [inline go.mod:7]: Maybe is better set `v1.22.4`?

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/google/btree v0.0.0-20161217183710-316fb6d3f031 // indirect
+	github.com/gregjones/httpcache v0.0.0-20170424210139-0d2297f241a3
+	github.com/miekg/dns v0.0.0-20170426174652-113c7538ea6d // indirect
+	github.com/peterbourgon/diskv v2.0.1-0.20160404093648-5dfcb07a075a+incompatible // indirect
+	github.com/registrobr/rdap v0.0.0-20160829181932-be455094b27e
```
Comment (rafaeljusto) [inline go.mod:13]: Would be great to define a tag in `github.com/registrobr/rdap` so we could set a version here.

Comment (hkkobayashi) [reply go.mod:13]: https://github.com/registrobr/rdap/releases/tag/v1.0.0

> For the `rdap-client` I think is better we keep the vendor package for now:
> 
> ```
> go mod vendor
> ```

I agree.

File: go.mod (hunk 1 above)
Comment (hkkobayashi) [reply go.mod:7]: v1.22.5 already and also update it to github.com/urfave/cli

File: go.mod (hunk 2 above)
Comment (hkkobayashi) [reply go.mod:13]: > Would be great to define a tag in `github.com/registrobr/rdap` so we could set a version here.

https://github.com/registrobr/rdap/releases/tag/v1.0.0

**Comment by hkkobayashi** [comment]
See https://github.com/registrobr/rdap-client/pull/8

**Pull request closed without merging**